manager/snapshot.go

```diff
@@ -0,0 +1,13 @@
+package manager
+
+import (
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+)
+
+func (m *VolumeManager) ListSnapshotCRsRO(volumeName string) (map[string]*longhorn.Snapshot, error) {
```

Comment:
Why not just rename `ListSnapshots` to `ListSnapshotInfos` and rename this to `ListSnapshots`? the same for `GetSnapshotCRRO`, because the naming is a little confusing.

Reply:
Updated. Thank you.